templates/v2/common/_button_with_loader.html.twig

```diff
@@ -1,3 +1,4 @@
+{% set text = text is defined ? text : 'submit'|trans %}
```

Comment:
I guess this is used in `_button_with_loader_content.html.twig` but should'nt we explicitly pass it to the component ? (it could be discussed as being a good practice)